diff -up ./tasks/task_diff_build.py.orig ./tasks/task_diff_build.py
--- ./tasks/task_diff_build.py.orig	2012-08-02 15:09:10.574253566 +0200
+++ ./tasks/task_diff_build.py	2012-09-14 07:44:34.104027365 +0200
@@ -29,7 +29,7 @@ class DiffBuild(TaskBase):
 
     def run(self):
         mock_config = self.args.pop("mock_config")
-        srpm_name = self.args.pop("srpm_name")
+        srpm_name = self.args.pop("srpm_name", None)
         keep_covdata = self.args.pop("keep_covdata", False)
         all_checks = self.args.pop("all", False)
         security_checks = self.args.pop("security", False)
